test/e2e/tests/send-hex-address.spec.js

```diff
@@ -123,43 +123,20 @@ describe('Send ERC20 to a 40 character hexadecimal address', function () {
     await withFixtures(
       {
         dapp: true,
-        fixtures: new FixtureBuilder()
-          .withPermissionControllerConnectedToTestDapp()
-          .build(),
+        fixtures: new FixtureBuilder().withTokensControllerERC20().build(),
         ganacheOptions,
         smartContract,
         title: this.test.title,
         failOnConsoleError: false,
       },
-      async ({ driver, contractRegistry }) => {
-        const contractAddress = await contractRegistry.getContractAddress(
-          smartContract,
-        );
+      async ({ driver }) => {
         await driver.navigate();
         await driver.fill('#password', 'correct horse battery staple');
         await driver.press('#password', driver.Key.ENTER);
-
-        await driver.openNewPage(
-          `http://127.0.0.1:8080/?contract=${contractAddress}`,
-        );
-        let windowHandles = await driver.getAllWindowHandles();
-        const extension = windowHandles[0];
-
-        // Using the line below to make wait time deterministic and avoid using a delay
-        // See more here https://github.com/MetaMask/metamask-extension/pull/15604/files#r949300551
-        await driver.findClickableElement('#deployButton');
-
-        // Add token
-        await driver.clickElement('#watchAsset');
-        await driver.waitUntilXWindowHandles(3);
-        windowHandles = await driver.getAllWindowHandles();
-        await driver.switchToWindowWithTitle(
-          'MetaMask Notification',
-          windowHandles,
-        );
-        await driver.clickElement({ text: 'Add token', tag: 'button' });
-        await driver.waitUntilXWindowHandles(2);
-        await driver.switchToWindow(extension);
+        await driver.waitForSelector({
+          css: '[data-testid="eth-overview__primary-currency"]',
+          text: '24.9977 ETH',
+        });
```

Comment:
Wait for the balance to load, else the test will be flaky, as the send screen would intermittently display the insufficient funds warning

Reply:
`You do not have enough ETH in your account to pay for transaction fees on Localhost 8545 network. Buy ETH or deposit from another account.`